docroot/sites/all/modules/features/forms/forms.module

```diff
@@ -1131,14 +1131,17 @@ function _forms_usage_page() {
   foreach ($result as $key => $block) {
     $usage_results = _ilr_get_block_usage($block->bid);
     if (count($usage_results)) {
-      $nid = $usage_results[0];
-      $alias = drupal_get_path_alias("node/$nid");
-      $content .= "<li><a href='/admin/structure/entityform_types/manage/$block->delta'>$block->delta</a>: <a href='/$alias'><strong>$alias</strong></a></li>\n";
+      // URL for managing the form
+      $content .= "<li><a href='/admin/structure/entityform_types/manage/$block->delta'><strong>$block->delta</strong></a><br>";
+      for ($i=0; $i<count($usage_results); $i++) {  // Nodes the form is on
```

Comment:
Thanks @carolineHecht. Please use a foreach loop rather than a for loop. For loops are rarely used in php, in part because it's expensive to call count on each iteration of the loop. Otherwise, looks good.